GUIDE.md

```diff
@@ -0,0 +1,331 @@
+# Running the Program
+
+### Configuration
+Before running the program you need to define CONFIG in config.py to include the settings and models you want.
+
+
+#### File Location
+```plaintext
+src/
+└── config.py
+```
+
+#### File
+
+Temperature and max_tokens should stay the default of 0.0 and 1024. If a model for whatever reason cannot use the defaults then define the individual model temperature and max_token fields to be as close as possible. If a model does not allow you to adjust it then record the value the company uses for the model.
+
+```python
+# ... Other Config
+
+# Runtime config
+# ADJUST THIS GLOBAL VARIABLE
+CONFIG = {
+    # Defines protocols to perform, done in seqence. Not all sequences are valid
+    "pipeline": [GET_SUMM, GET_JUDGE, GET_RESULTS],
+    # Specifies if summary jsonl should be overwritten before appending new data. Judgemennts and results are always overwritten before appending
+    "overwrite": True,
+    # Specifies the article dataset to generate summaries for
+    "input_file": DATA_PATH,
```

Comment:
better to rename it to "source_article_path" and please use the path to the actual leaderboard source article file -- we should have it as a CSV/JSON file in the repo, right?